Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/Controller.purs

```diff
@@ -894,7 +901,7 @@ eval (UpdateSource lat lng name) state = do
 
 eval (HideLiveDashboard val) state = continue state {props {showLiveDashboard =false}}
 
-eval LiveDashboardAction state = do
+eval (DriverInfoCardActionController (DriverInfoCardController.LiveDashboardAction)) state = do
```

Comment:
normal dashboard flow will not work if we do like this. create separate eval of this and call `LiveDashboardAction`inside it.